pkg/transaction/metric/metricspublisher.go

```diff
@@ -67,13 +68,30 @@ func (c *metricPublisher) publishToLighthouse(event LighthouseUsageEvent) error
 		c.logger.WithError(err).Error("publishing usage")
 		return err
 	}
-	if response.Code >= 400 {
-		resBody := string(response.Body)
+
+	if response.Code < 400 {
+		return nil
```

Comment:
Should this look specifically for the 202 response code and not just under 400? That way we get some sort of log when its an unexpected response, likely in the unmarshal error block.